src/components/shared/Bandeau/Bandeau.tsx

```diff
@@ -0,0 +1,36 @@
+'use client'
+
+import { ReactElement, useContext } from 'react'
+
+import styles from './Bandeau.module.css'
+import { clientContext } from '../ClientContext'
+
+export interface BandeauInformations {
+  titre: string | undefined
+  description: string | undefined
+}
```

Comment:
```suggestion
type BandeauInformations = Readonly<{
  titre: string | undefined
  description: string | undefined
}>
```

Ce type ne devrait pas être déclaré et exporté ici.